languages.js

```diff
@@ -0,0 +1,44 @@
+exports.english = {
```

Comment:
I wouldn't rename it to default. This is the language pack in english. I would add that `logic` in the languageChooser (or whatever is named).